distribution/src/resources/log4j2.xml

```diff
@@ -55,7 +55,7 @@
 
         <RollingFile name="all-out" fileName="${sys:openfireHome}/logs/all.log" filePattern="${sys:openfireHome}/logs/all.log-%i">
             <PatternLayout>
-                <Pattern>%d{yyyy.MM.dd HH:mm:ss} %-5p [%t]: %c - %msg%n</Pattern>
+                <Pattern>%d{ABSOLUTE} %highlight{%-5p} [%15.15tid]: %c{1.40} - %msg%n</Pattern>
```

Comment:
This seems unrelated to the cod change - is it deliberate?

Reply:
No, this is an artifact of us working with log files a lot, and optimizing them a little for our sanity. This change shouldn't be in this PR.

I have already raised a PR with a similar change here: https://github.com/igniterealtime/Openfire/pull/1874